docs/guides/update-guide/120-to-130.md

```diff
@@ -17,6 +17,35 @@ A critical [issue](https://github.com/camunda-cloud/zeebe/issues/8611) which may
 Please refer to the [release notes](https://github.com/camunda-cloud/zeebe/releases/tag/1.3.2) for more.
 :::
 
+With Zeebe 1.3.0, we removed the monitoring specific configuration from the broker and the gateway. Instead, the monitoring configuration is now completely handled via the Spring Boot management server. You can read [this page](https://docs.spring.io/spring-boot/docs/2.6.x/reference/htmlsingle/#actuator) to find out more how to configure the server.
+
+For the broker, this replaces the following configuration settings:
+
+```yaml
+zeebe:
+  broker:
+    network:
+      monitoringApi: # everything in this section
```

Comment:
:wrench: Just a nitpick but maybe just add the host and port values here instead of saying "everything in this section". I had to look up our old configs because I wasn't sure if 1.2 supported more than host and port which are no longer supported in 1.3.

Same for the gateway config.